src/components/ChartContextMenu.svelte

```diff
@@ -152,12 +154,18 @@
           </a>
         </div>
       </div>
-      <div class="option">
-        <div class="option-icon" />
-        <div class="option-link">
-          <i>Coming soon: PUSHLOG LINK</i>
+      {#if pushlogUrl}
+        <div class="option">
+          <div class="option-icon">
+            <a href={pushlogUrl} target="pushlog">
+              <GitBranch size="12" />
+            </a>
+          </div>
+          <div class="option-link">
+            <a href={pushlogUrl} target="pushlog">View Changesets in Range</a>
```

Comment:
What do you think about using a more definitive name like "View Pushlog Range" for this feature, instead of "View Changesets in Range"?

Reply:
That doesn't sound accurate to me. The link will take you to a page that lists all changesets between the two revisions (selected from the build ID range).